src/Arcus/Utilities/SubnetUtilities.cs

```diff
@@ -12,14 +12,46 @@ namespace Arcus.Utilities
     /// </summary>
     public static class SubnetUtilities
```

Comment:
Why SubnetUtilities and not IPAddressRanageUtitilies? I can argue for either but am curious about your thinking.

Reply:
Simply because IANNA defines the private address ranges as a CIDR and and using a `Subnet` seemed to be the most straight forward way to express it. Not only is it one to one with the docs it prevents me from introducing an error by pulling out the head and tail manually. At the end of the day they both implement `IIPAddressRange`.